Source/Bogus/Randomizer.cs

```diff
@@ -152,6 +221,16 @@ public decimal Decimal(decimal min = 0.0m, decimal max = 1.0m)
          return Convert.ToDecimal(Double()) * (max - min) + min;
       }
 
+      /// <summary>
+      /// Generate a random decimal, based on the specified normal distribution.
+      /// </summary>
+      /// <param name="mean">Mean average of the normal distribution</param>
+      /// <param name="standardDeviation">Standard deviation of the normal distribution</param>
+      public decimal DecimalFromNormalDistribution(double mean, double standardDeviation)
```

Comment:
Also, it would be really helpful if you could include an example of the typical usage of your method. Just so people don't go off throwing numbers crazy nubmers as parameters. Something like:
```
/// <summary>
/// Generate a random decimal, based on the specified normal distribution.
/// For example, to create random values around an average height of 69.1
/// inches with a standard deviation of 2.9 inches away from the mean
/// call DecimalFromNormalDistribution(69.1, 2.9).
/// </summary>
/// <param name="mean">Mean average of the normal distribution</param>
/// <param name="standardDeviation">Standard deviation of the normal distribution</param>
```
I usually find it really helpful when XML docs have a small example use of how to use the method.